### ISSUE TITLE
h1analysis with distRDF

### ISSUE THREAD

**Issue by Axel-Naumann** [root-project/root#8985]
### Explain what you would like to see improved

We should be able to replace the Proof version of the h1analysis tutorial with one using distRDF. The tutorial should work without any external scheduler, i.e. "just" local multi-process to demonstrate what a distRDF analysis looks like.

Once distRDF is non-experimental, the proof tutorial should be moved to legacy/.

**Comment by etejedor** [comment]
Ok for translating the tutorial to DistRDF 👍 

About the local multi-process execution: this implies we'd need to create a local backend that spawns local Python processes. Is that something we want, since we can just fall back to regular MT RDF for local runs?

**Comment by Axel-Naumann** [comment]
Having local multiprocess distRDF might be really useful also for testing?

**Comment by etejedor** [comment]
Yes, perhaps just for testing? It would allow to test the creation of ranges and the task processing in separate worker processes with no external scheduler (now we can do the same local runs with Spark and Dask but we need the corresponding libraries). Performance-wise still MT RDF should be the best.

**Comment by dpiparo** [comment]
Reassigning to @martamaja10

**Comment by martamaja10** [comment]
After a careful consideration we decided to close this item in particular - i.e. we will not proceed with the H1 Analysis translation to distRDF. Instead, we will focus on providing a distRDF tutorial with the recently published CMS OpenData (https://opendata.cern.ch/docs/cms-releases-2016data-2024) as we believe such a tutorial will serve the community best.
